docs/install.rst

```diff
@@ -113,6 +113,41 @@ starting by installing the `Anaconda Distribution
 Mac, Windows) and installs a full-featured scientific Python in a user directory
 without requiring root permissions.
 
+.. _anaconda_install:
+
+Using Conda
+-----------
+
+To install ``astropy`` using conda run::
+
+    conda install astropy
+
+``astropy`` is installed by default with the `Anaconda Distribution
+<https://www.anaconda.com/distribution/>`_. To update to the latest version run::
+
+    conda update astropy
+
+There may be a delay of a day or two between when a new version of ``astropy``
+is released and when a package is available for conda. You can check
+for the list of available versions with ``conda search astropy``.
+
+It is highly recommended that you install all of the optional dependencies with::
+
+    conda install -c astropy -c defaults \
+      scipy h5py beautifulsoup4 html5lib bleach pyyaml pandas sortedcontainers \
+      pytz matplotlib setuptools mpmath bottleneck jplephem asdf
+
+To also be able to run tests (see below) and support :ref:`builddocs` use the
+following. We use ``pip`` for these packages to ensure getting the latest
+releases which are compatible with the latest ``pytest`` and ``sphinx`` releases::
```

Comment:
I think these two can now be included with the `conda install` command above. While there is a lag of a couple of hours between when the package is published on pypi and when it shows up in conda-forge it isn't much more than that these days.  Any advantage gained by installing these with `pip` is outweighed, I think, by the disadvantage of having to remember which things are upgraded by `pip` and which by `conda`.

Reply:
Hi @cosmictiger! Are you still working on this?